Bottom: 874eff7842dbdeb251567b9b0eda2345571b910d
Top:    8b3e2041f87e7886c74b322d3de8e8857c7f0b24
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-11 11:23:16 +0800

Refresh of net-core-rename

---

diff --git a/net/core/dev.c b/net/core/dev.c
index 7e6a2f6..b7eb26d 100644
--- a/net/core/dev.c
+++ b/net/core/dev.c
@@ -4129,9 +4129,9 @@ static struct netdev_rx_queue *netif_get_rxqueue(struct sk_buff *skb)
 	return rxqueue;
 }
 
-static u32 netif_receive_generic_xdp(struct sk_buff *skb,
-				     struct xdp_buff *xdp,
-				     struct bpf_prog *xdp_prog)
+static u32 do_xdp_generic_core(struct sk_buff *skb,
+			       struct xdp_buff *xdp,
+			       struct bpf_prog *xdp_prog)
 {
 	struct netdev_rx_queue *rxqueue;
 	void *orig_data, *orig_data_end;
@@ -4259,7 +4259,7 @@ int do_xdp_generic(struct bpf_prog *xdp_prog, struct sk_buff *skb)
 		u32 act;
 		int err;
 
-		act = netif_receive_generic_xdp(skb, &xdp, xdp_prog);
+		act = do_xdp_generic_core(skb, &xdp, xdp_prog);
 		if (act != XDP_PASS) {
 			switch (act) {
 			case XDP_REDIRECT:
